src/core/templates/partials/component.jsx

```diff
@@ -0,0 +1,59 @@
+import Adapt from 'core/js/adapt';
+import { compile, classes, helper, html } from 'core/js/reactHelpers';
+
+export default function(model, view) {
+  const data = model.toJSON();
+  data._globals = Adapt.course.get('_globals');
+  // Create references to un-controlled view containers
+  view.jsxHeading = view.jsxHeading || React.createRef();
+  view.jsxComponentDescription = view.jsxComponentDescription || React.createRef();
+  const {
+    displayTitle,
+    body,
+    instruction,
+    mobileInstruction,
+    _component,
+    _disableAccessibilityState
+  } = data;
+  const type = _component.toLowerCase();
+  const sizedInstruction = (mobileInstruction && Adapt.device.screenSize !== 'large') ?
+    mobileInstruction :
+    instruction;
+  return (displayTitle || body || sizedInstruction) &&
```

Comment:
Please wrap the template body in parentheses i.e.
```js
return (displayTitle || body || sizedInstruction) && (
  // ...
);
```

Reply:
Done

Reply:
Misssing the semi-colon.